front-end/src/util/__tests__/getEncodedAddress.test.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import getEncodedAddress from '../getEncodedAddress';
+
+jest.mock('../getNetwork', () => jest.fn(() => {return 'polkadot';}));
+const getNetwork = require('../getNetwork');
+
+describe('Verify address is encoded', () => {
+	beforeEach(() => {
+		jest.resetModules();
+	});
+
+	const address1 = 'FqfW96FuXRH7pRy9njeNSxu4CNoBM1hKnVAmNRrFrU2izwj';
+
+	it('for Kusama', () => {
+		getNetwork.mockImplementation(() => 'kusama');
+		expect(getEncodedAddress(address1)).toBe('FqfW96FuXRH7pRy9njeNSxu4CNoBM1hKnVAmNRrFrU2izwj');
+		expect(getEncodedAddress(address1)).not.toBe('A41F9YrXEYYxm2WFFrczYhUQiEW8APB8qw39gnYnzsYSZ9M');
```

Comment:
test `FOO === BAR` and `FOO !== BAZ` is the same as `FOO === BAR` unless `FOO` is schizophrenic.
TL;DR, this second test doesn't bring any info.

edit: ah, now I understand why you have this second test. In this case, using variabled would make sense:
```js
const KUSAMA_ENCODED = 'Fq...'
const POLKADOT_ENCODET = 'A41..'

test for kusama:
-  KUSAMA_ENCODED => KUSAMA_ENCODED
-  POLKADOT_ENCODED => KUSAMA_ENCODED

test for polkadot 
POLKADOT_ENCODED => POLKADOT_ENCODED
KUSAMA_ENCODED => POLKADOT_ENCODED
```